app/routes/customer_routes.py

```diff
@@ -0,0 +1,153 @@
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from flask import Blueprint, jsonify, request, make_response
+import requests
+from datetime import datetime, timedelta
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+
+# refactored and cleaned
+
+
+@customers_bp.route("", methods=["POST"])
+def create_customer():
+    request_body = request.get_json()
+
+    if "name" not in request_body:
+        return jsonify({"details": "Request body must include name."}), 400
+    if "postal_code" not in request_body:
+        return jsonify({"details": "Request body must include postal_code."}), 400
+    if "phone" not in request_body:
+        return jsonify({"details": "Request body must include phone."}), 400
+
+    new_customer = Customer(
+        name=request_body["name"],
+        postal_code=request_body["postal_code"],
+        phone=request_body["phone"],
+    )
+    db.session.add(new_customer)
+    db.session.commit()
+
+    response_body = {
+
+
+        "id": new_customer.customer_id,
+        "name": new_customer.name,
+        "postal_code": new_customer.postal_code,
+        "phone": new_customer.phone,
+
+    }
+
+    return jsonify(response_body), 201
+
+
+@customers_bp.route("", methods=["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customer_response = []
+
+    for customer in customers:
+        customer_response.append({
+            "id": customer.customer_id,
+            "name": customer.name,
+            "registered_at": customer.registered_at,
+            "postal_code": customer.postal_code,
+            "phone": customer.phone,
+        })
```

Comment:
All of the classes could utilize a helper function to return instance data into dictionaries. In this case, a helper function would have reduced these 5 lines of code into 1. 

```suggestion
        customer_response.append(customer.to_dict())
```